python/ray/autoscaler/autoscaler.py

```diff
@@ -265,6 +265,7 @@ def reload_config(self, errors_fatal=False):
             with open(self.config_path) as f:
                 new_config = yaml.safe_load(f.read())
             validate_config(new_config)
+            new_config.setdefault("cluster_synced_files", [])
```

Comment:
I think there's another place you can set this (i.e. in the json schema, marking this field as optional?)

Reply:
This field is optional in the json schema, it's just expected to exist in the implementation because it assumes the optional field gets filled with a default value